code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -516,6 +522,11 @@
 		"is stabbing themselves with their mandibles!",
 		"is holding their breath!")
 
+/datum/species/kidan/handle_death(var/mob/living/carbon/human/H)
```

Comment:
This will be problematic for anyone who has this organ is is a non-Kidan, as they'll continue to glow, after death.

Does give me an idea though...